examples/features/dns/README.md

```diff
@@ -21,16 +21,6 @@ NAMESPACE=($(kubectl create -f ../../use-cases/namespace.yaml)[0])
 NAMESPACE=${NAMESPACE:10}
 ```
 
-2. Register namespace in `spire` server:
```

Comment:
Please correct step numbers for the next steps.

Reply:
ok, done